.github/workflows/ubiquibot-config.yml

```diff
@@ -0,0 +1,39 @@
+---
+baseMultiplier: 1000
```

Comment:
This is useful to modify price calculations. All of the prices are a function with this as the "base" constant.

e.g. `Priority: 0 (Normal)` and `Time: <1 Day` I believe yields `100 USD` with this base rate.